src/main/java/bio/terra/service/profile/ProfileRequestValidator.java

```diff
@@ -27,12 +27,6 @@ public boolean supports(Class<?> clazz) {
   public void validate(@NotNull Object target, Errors errors) {
     if (target != null && target instanceof BillingProfileRequestModel) {
       BillingProfileRequestModel billingProfileRequestModel = (BillingProfileRequestModel) target;
-      String billingAccountId = billingProfileRequestModel.getBillingAccountId();
-      if (billingAccountId == null || !isValidAccountId(billingAccountId)) {
-        errors.rejectValue(
-            "billingAccountId",
-            "The id must be 3 sets of 6 capitalized alphanumeric characters separated by dashes");
-      }
```

Comment:
Should we instead check that either a billing profile or a storage account is specified? And I would think we still want to validate the billing account id if it is provided.

Reply:
I moved this validation to line 75, so we validate the billing account id for GCP billing profiles. I think the checks in `isValidCloudPlatform` are sufficient...what do you think?

Reply:
Oh, I misread what this validation was originally doing. This all looks good to me!